### PR TITLE
Update secure-wallet doc

### PR DESCRIPTION
**Pull request by m52go**
Updates, summarized:
- specify Bisq wallet details
- clarify that BTC and BSQ wallets use the same seed
- inform that restoring from seed is a last-resort measure and offer alternatives
- update screenshots

I adjusted the title of the doc to reflect its broader scope.

### CODE REVIEW

**Comment by m52go** [comment]
@erciccione would you mind reviewing this?

Also, would you be okay with being added to the bisq-network organization, so that you can be assigned as a reviewer?

**Comment by erciccione** [comment]
I will review this and (at least) #165 tomorrow (Europe).

> Also, would you be okay with being added to the bisq-network organization, so that you can be assigned as a reviewer?

Sure :)

**Comment by m52go** [comment]
@erciccione thank you! You should be receiving an invite soon.

File: secure-wallet.adoc [hunk 1]
```diff
@@ -15,17 +25,27 @@ To set a password, go to the `Wallet password` section of the `Account` screen a
...
 
 .Write down your Bisq wallet's seed words & wallet date.
 image::seed-words.png[Write down your Bisq wallet's seed words]
 
-With your wallet encrypted and your seed words written down, the bitcoin in your Bisq wallet is much more secure. Bravo!
+With your wallet encrypted and your seed words written down, the bitcoin in your Bisq wallet is much more secure.
+
+== Restoring from seed
+
+WARNING: Your seed words can restore your wallet's BTC and BSQ, but they *do not* restore other Bisq application data such as your payment accounts, open disputes, etc. For that, you need to <<backup-recovery#, back up the Bisq data folder>>.
```
Comment (erciccione) [inline secure-wallet.adoc]: Maybe worth to add that also "Reputation BSQ" are not transferred

**Pull request merged**